symplyphysics/laws/electricity/circuits/waveguides/wavelength_in_rectangular_waveguide.py

```diff
@@ -1,34 +1,44 @@
+"""
+Wavelength in rectangular waveguide
+===================================
+
+A rectangular waveguide is a rectangular metal waveguide capable of supporting waves
+propagating along it. There is a critical wavelength. Signals with a wavelength greater
+than the critical one are attenuated and do not propagate in the waveguide.
+
+"""
+
 from sympy import Eq, solve, sqrt
 from symplyphysics import (
-    units,
     Quantity,
-    Symbol,
-    print_expression,
     validate_input,
     validate_output,
+    symbols,
+    clone_as_symbol,
 )
 
-## Description
-## A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
-## There is a critical wavelength. Signals with a wavelength greater than the critical one are attenuated and
-## do not propagate in the waveguide.
+waveguide_wavelength = clone_as_symbol(symbols.wavelength, subscript="\\text{w}")
```

Comment:
See `Guide wavelength` section here: https://www.microwaves101.com/encyclopedias/waveguide-mathematics
Explain what is waveguide wavelength and what is signal wavelength.